Realm/ObjectServerTests/RealmServer.swift

```diff
@@ -684,25 +686,110 @@ public class RealmServer: NSObject {
             "relationships": [:]
         ]
 
-        let userDataRule: [String: Any] = [
+        let uUIDPrimaryKeyObjectRule: [String: Any] = [
```

Comment:
`uuidPrimaryKeyObjectRule`

Initialisms always have the same capitalization for the entire thing. In obj-c they're always all uppercase unless they're at the start of a local variable name. In Swift they're uppercase unless they're at the start of any non-type name.